manifests/u/UbiquitiInc/IdentityDesktop/0.1.13.31/UbiquitiInc.IdentityDesktop.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: UbiquitiInc.IdentityDesktop
+PackageVersion: 0.1.13.31
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.10240.0
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{0163E59F-332B-4936-86C7-A664306728E9}'
+ReleaseDate: 2024-01-17
+Installers:
+- InstallerUrl: https://fw-download.ubnt.com/data/uid-identity-standard-desktop-app-msi/0d0e-windows-0.1.13-a482dd66-ff91-4835-bf2c-d32fe29f352f.msi
```

```suggestion
- InstallerUrl: https://fw-download.ubnt.com/data/uid-identity-standard-desktop-app-msi/49b8-windows-0.1.13-3377712d-a3ed-498b-bcdc-d56b76783066.msi
```
I got a slightly different path and GUID.

---

Touché. Must've been updated in the meantime.